pyensembl/genome.py

```diff
@@ -438,7 +442,11 @@ def transcript_sequence(self, transcript_id):
         if self.transcript_sequences is None:
             raise ValueError(
                 "No transcript FASTA supplied to this Genome: %s" % self)
-        return self.transcript_sequences.get(transcript_id)
+        sequence = [x for x in map(lambda x: x.get(transcript_id), self.transcript_sequences) if x is not None]
+        #self.transcript_sequences.get(transcript_id)
```

Comment:
This was intentional. My latest code went back to how this code section originally was, though